_data/fa.yml

```diff
@@ -0,0 +1,361 @@
+    </ol>
+  hReputationSystem: "آیا یک سیستم اعتباری خالص میان معامله گران وجود داوران را غیرضروری نمی کند؟"
+  pReputationSystem: >
+    <p>شهرت خالص بدون گزینه داوری یک سیستم محافظت ضعیف است، زیرا شما هرگز نمی توانید از <a href="https://en.wikipedia.org/wiki/Sybil_attack" target="_blank">Sybil</a> یا <a href="https://en.wikipedia.org/wiki/Confidence_trick" target="_blank">حملات طولانی</a> مدت جلوگیری کنید.</p> <p>این مکانیسم ها نیز برای حفاظت از حریم خصوصی و حفظ خدمات غیرمتمرکز دارای مشکل هستند.</p>
+  hBeArbitrator: "من می توانم یک داور باشم؟"
+  pBeArbitrator: "<p>ضروری <a href=\"https://docs.bisq.network/dao/phase-zero.html\" target=\"_blank\" rel=\"noopener\">BSQ bond</a> بسیار بالا مورد نیاز است و در حال حاضر تنها تعداد کمی از افرادی که به اندازه کافی به Bisq کمک کرده اند</p>"
+  hContactPeer: "آیا می توانم با همکار تجاری خود تماس بگیرم؟"
+  pContactPeer: >
+    <p>تا کنون، Bisq عمدا از ارائه یک راه برای برقراری ارتباط مستقیم با هخمکاران تجاری خود اجتناب کرده است. به طور کلی نیازی به چنین مکانیزمی ندارید و پیشنهاد می کنیم به این موارد امنیتی توجه کنید: کاربران مخرب می توانند بدون استفاده از Bisq وارد یک تجارت مستقیم شوند و سپس از فقدان امنیت استفاده کنند. شخص دیگر، یا از کانال مستقیم برای سایر فعالیت های جعلی (کلاهبرداریهای اجتماعی مهندسی، هرزنامه و غیره) استفاده می کنید.</p> <p>در عوض، در صورت هر گونه مسائل، کاربران می توانند با کمک یک داور اختلافات را باز و حل و فصل نمایند. همچنین برای دسترسی به سایت ها و <a href='https://bisq.community' target='_blank'>انجمن های Bisq </a> و <a href='https://bisq.network/slack-invite' target='_blank'>کانال های Slack</a> در دسترس هستند.</p> <p>نکته: با راه اندازی <a href='https://github.com/bisq-network/proposals/issues/52' target='_blank'>پروتکل تجاری جدید </a> ,با این حال، یک ابزار برای همکارانی وجود دارد که بتوانند به طور مستقیم با یکدیگر ارتباط برقرار کنند.</p>
+  hBisqDAO: The Bisq DAO
+  hBisqDaoWhat: "BISQ DAO چیست؟"
+  pBisqDaoWhat: >
+    <p>BISQ DAO (سازمان خودمختار غیر متمرکز) یک مکانیزم اداری غیرمتمرکز برای نرم افزار Bisq است. که برای یک شرکت است. اگر یک نرم افزار غیر متمرکز توسط یک نهاد واحد مثل یک شرکت کنترل شود, چقدر می تواند مفید باشد؟</p> <p>یک نرم افزار غیر متمرکز بدون بدون حکمرانی غیر متمرکز مفید واقع نخواهد بود.</p> <p>به همین دلیل است که Bisq یک شرکت یا نهاد قانونی از هر نوعی نیست - بلکه به عنوان یک DAO سازماندهی شده است. DAO برنامه های مالی و استراتژی های نرم افزاری را مدیریت می کند تا شبکه Bisq را تقویت کند و در عین حال آن را در برابر حملات به زیرساخت هایی که رهبری و عملیات آن را کنترل می کند، سخت می کند.</p> <p>به طور خلاصه، Bisq DAO قادر است Bisq را <a href='https://docs.bisq.network/intro.html' target='_blank'>از همان ابتدا به یک اصل اصلی پروژه</a> تبدیل کند که بیشتر مقاوم در برابر سانسور باشد.</p> <p>در این مقدمه، بیشتر درباره Bisq DAO بخوانید.<a href='https://docs.bisq.network/user-dao-intro.html' target='_blank'>.</p>
```

Comment:
You open a hyperlink at the end of this string, but you don't close. This is messing up the format of the section that follows. I removed the link locally while testing and everything worked fine.

Reply:
Thanks, @erciccione. After some searching I've found it yesterday. @Blockchainsmoker has taken care of it and I'll update it soon.